### ISSUE TITLE
Email notification issues with r1295 from r1070 (upgrade)

### ISSUE THREAD

**Issue by chewy-chewbacca** [projectsend/projectsend#848]
Again Ignacio, thank you for such a wonderful open-source project! 👍 

We upgraded from r1070 to r1295 and route our notification email via Gmail (G-suite). With the new version the email test within PS works just fine, but our admins and explicit emails defined in the email notifications options are not receiving emails. I checked the sending Gmail account and I see the tests in there but it seems notifications are never hitting Google at all. I even checked the mail log in G-Suite and there were no emails from our ProjectSend other than the tests. 

Any ideas in terms of troubleshooting?

**Comment by ignacionelson** [comment]
Thanks for your comments! Glad you find PS useful!
Does this happen just for notifications or is it also happening on other email types too? (recover password, new account welcome, etc).

**Comment by chewy-chewbacca** [comment]
Bizarre. I started testing the password reset, new user which worked. Tested further and now everything works. Disregard!

**Comment by chewy-chewbacca** [comment]
Opening this back up. It seems that all existing users on our system don't trigger any notifications. When I tested the above I created a new user and that user received all emails.

**Comment by chewy-chewbacca** [comment]
I should say existing "clients". If I create a new client they trigger alerts etc., but all our old users that existed pre-upgrade do not. I looked at the database and couldn't find any notable differences in the user table between the old and new users. The only difference I could find is that new clients with no data in address, phone and contact are set to "null" whereas the existing users are not set to "null".

**Comment by ignacionelson** [comment]
This just keeps getting weirder!
Does this happen for system users too? Have you tried all types of notifications? eg: recover password.

I believe this could be tested by doing a var_dump on the email object before it's sent, to see if it gets populated

**Comment by chewy-chewbacca** [comment]
As far as I can tell recover password works across the board, I just tested it with an existing (not new) system user. As a system user I also tried uploading files with the "send copies if a system user uploads files" checked and used my system user to upload a file. No notification was sent in this instance. 

The issue, from what I can tell only affects old clients and (possibly) users too. 

Let me know how I do a var_dump or any troubleshooting steps from here you think might be helpful.
